fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/api/IDatabaseTranslator.java

```diff
@@ -166,6 +166,16 @@
      */
     String limit(String arg);
 
+    /**
+     * Construct a pagination clause specific to the type of database
+     * To be deterministic, queries using pagination must include an
```

Comment:
```suggestion
     * @implNote To be deterministic, queries using pagination must include an
```

Seems like an `@implNote`

Reply:
ok